### DISCUSSION TITLE
WIederherstellungsphrase anzeigen

### DISCUSSION THREAD

**Discussion by PG79Linz** [iotaledger/firefly#8617] [Help & Support]
Hallo Leute. Gibt es eine Möglichkeit sich die Wiederherstellungsphrase auf FireFly neu anzeigen zu lassen? habe mir damals nur (warum auch immer?) nur einen Teil davon aufgeschrieben. komischerweise genau 12 der 24 Wörter.... egal

Git es da eine Möglichkeit?

**Comment by begonaalvarezd** [comment]
That is not possible, stronghold is designed so that the 24 words mnemonic can never be extracted. If you lost your 24 words you can create a new profile and transfer your funds there, this time making sure to write down the full correct recovery phrase
